paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -163,6 +163,12 @@ class PaparazziPlugin : Plugin<Project> {
         test.outputs.dir(snapshotOutputDir)
 
         val paparazziProperties = project.properties.filterKeys { it.startsWith("app.cash.paparazzi") }
+        test.systemProperties.putAll(paparazziProperties)
+
+        // Explicitly register these as inputs so that they are considered when determining up-to-date.
+        // The properties become resolvable after the last afterEvaluate runs.
+        test.inputs.property("paparazzi.test.record", isRecordRun)
+        test.inputs.property("paparazzi.test.verify", isVerifyRun)
```

Comment:
style nit: can we move this under the other "inputs" (just after "paparazzi.nativePlatform")?  also, let's remove the comment, since this is standard Gradle knowledge.